client/app/core/session.service.js

```diff
@@ -129,9 +139,9 @@ export function SessionFactory ($http, $sessionStorage, $cookies, RBAC, Polling)
   }
 
   // Helpers
-
   function active () {
-    // may not be current, but if we have one, we'll rely on API 401ing if it's not
-    return angular.isString(model.token) ? model.token : false
+    const token = vm.token.token || false
```

Comment:
why would it be not defined? your state should have a default state avoid this edge case.